ape_foundry/provider.py

```diff
@@ -427,6 +431,8 @@ def build_command(self) -> List[str]:
             "--steps-tracing",
             "--block-base-fee-per-gas",
             f"{self.settings.base_fee}",
+            "--gas-price",
+            f"{self.settings.gas_price}",
```

Comment:
I believe you can't have both base fee and gas price at the same time

To get a more consistently nice testing environment, I think a gas price of zero works for local testing. For fork testing, I'm not sure if that'll work but it is better than using base fee because that always requires deducting gas

Reply:
i think it is just 1 settings applies to 1 type of tx and the other applies to another type of tx
also seems like brownie uses pre-1559 txns by default, which is how i got this bug... because i was trying to copy brownie in ape

Reply:
Yes, it went out of favor around the time EIP-1559 got popular